source/common/runtime/runtime_features.cc

```diff
@@ -61,6 +61,7 @@ RUNTIME_GUARD(envoy_reloadable_features_strict_check_on_ipv4_compat);
 RUNTIME_GUARD(envoy_reloadable_features_support_locality_update_on_eds_cluster_endpoints);
 RUNTIME_GUARD(envoy_reloadable_features_test_feature_true);
 RUNTIME_GUARD(envoy_reloadable_features_udp_listener_updates_filter_chain_in_place);
+RUNTIME_GUARD(envoy_reloadable_features_unique_extension_type_url);
```

Comment:
The patch looks good, but I am a little bit confused by the name `unique`. Do you think something like `prefer` would be better?

Reply:
Makes sense, renamed.